G/gb/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "gb"
+version = v"0.17.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/ederc/gb.git", "ec98ed8deeeb47e3e7bdf38eb947453bfa6b23b5")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd gb/
+./autogen.sh 
+./configure --enable-shared --disable-static --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-gmp=${prefix}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("x86_64", "freebsd"; )
+]
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libgb--no-undefined", :libgb)
```

Comment:
:flushed:

Reply:
Don't ask me. Is this a problem?

Reply:
This is a very, very strange library name.  It looks to me like the library name should be `libgb`, but a compiler flag (`--no-undefined`) has somehow gotten mashed into the library name.

Reply:
Looking into it.